docs/accuracy/accuracy-benchmarking.md

```
@@ -98,38 +98,30 @@ The env var is read at every `load_problems` call (no module-reload needed)
+usage. `trust_remote_code=True` is required because LCB still ships a
+repository loading script; `datasets<4` runs it normally. Nothing is
+bundled with the aiperf wheel — all subsets are fetched on-demand and
+cached under `~/.cache/huggingface/datasets/`.
+
+**Compatibility:** `livecodebench/code_generation_lite` requires
+`datasets<4`. `datasets>=4` dropped support for repository loading scripts
+entirely, and the loader surfaces a clear error when it detects this:
 
-```bash
-export AIPERF_ACCURACY_LCB_RELEASE_TAG=v6   # or whatever LCB now ships
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)